content/docs/lists-and-keys.md

```diff
@@ -268,7 +268,7 @@ In the examples above we declared a separate `listItems` variable and included i
 ```js{3-6}
 function NumberList(props) {
   const numbers = props.numbers;
-  const listItems = numbers.map((number) =>
+  const listItems = numbers.map((number) => (
```

Comment:
This is valid:
```js
const numbers = [1, 2, 3, 4, 5];
const listItems = numbers.map((number) =>
  <li>{number}</li>
);
```
The parens are balanced.

The proposed change would be invalid though.

Maybe it helps if you add newlines:
```js
const numbers = [1, 2, 3, 4, 5];
const listItems = numbers.map(
  (number) => ( // New opening paren...
    <li>{number}</li>
  // But no closing paren
);
```

Reply:
Thanks for the explanation @bvaughn , sorry for my mistake..

Reply:
No worries!